src/Microsoft.SqlTools.ServiceLayer/Connection/Contracts/ConnectionCompleteNotification.cs

```diff
@@ -52,6 +52,16 @@ public class ConnectionCompleteParams
         /// The type of connection that this notification is for
         /// </summary>
         public string Type { get; set; } = ConnectionType.Default;
+
+        /// <summary>
+        /// Gets or sets a boolean value indicates whether the current server version is supported by the service.
+        /// </summary>
+        public bool IsSupportedVersion { get; set; }
+
+        /// <summary>
+        /// Gets or sets the additional warning message about the unsupported server version.
+        /// </summary>
+        public string UnsupportedVersionMessage { get; set; }
```

Comment:
Not sure why you added this if you're not using it here (and it's pretty unlikely to ever be used with how things are now...)

Reply:
as I mentioned in the description, I am working with Alan Yu on this, hopefully next week sometime we will have the web page.